service_mesh/service_mesh_install/threescale-adapter.adoc

```diff
@@ -0,0 +1,26 @@
+[id="threescale-adapter"]
+= Using the 3scale Istio adapter
+include::modules/ossm-document-attributes.adoc[]
+:context: threescale-adapter
+toc::[]
+
+The 3scale Istio Adapter allows you to label a service running within the {ProductName} and integrate that service with the 3scale API Management solution.
+
```

Comment:
Do you have a concept about 3scale and why you'd want to use it? If so, I'd put it in this assembly. If not, it might be a good RFE.

Reply:
Writing 3scale content is not part of our scope. I will leave this to @dfennessy from the 3scale team.